cpufreq/src/cpufreq-selector/cpufreq-selector.c

```diff
@@ -23,143 +23,145 @@
+    return error_quark;
 }
 
 static void
 cpufreq_selector_init (CPUFreqSelector *selector)
 {
 
-	selector->priv = cpufreq_selector_get_instance_private (selector);
+    selector->priv = cpufreq_selector_get_instance_private (selector);
 
-        selector->priv->cpu = 0;
+    selector->priv->cpu = 0;
 }
 
 static void
 cpufreq_selector_class_init (CPUFreqSelectorClass *klass)
 {
-        GObjectClass *object_class = G_OBJECT_CLASS (klass);
-
-        object_class->set_property = cpufreq_selector_set_property;
-        object_class->get_property = cpufreq_selector_get_property;
-
-	/* Public virtual methods */
-	klass->set_frequency = NULL;
-	klass->set_governor = NULL;
-
-	/* Porperties */
-        g_object_class_install_property (object_class,
-					 PROP_CPU,
-                                         g_param_spec_uint ("cpu",
-							    NULL,
-							    NULL,
-                                                            0,
-							    G_MAXUINT,
-							    0,
-							    G_PARAM_CONSTRUCT_ONLY |
-                                                            G_PARAM_READWRITE));
+    GObjectClass *object_class = G_OBJECT_CLASS (klass);
+
+    object_class->set_property = cpufreq_selector_set_property;
+    object_class->get_property = cpufreq_selector_get_property;
+
+    /* Public virtual methods */
+    klass->set_frequency = NULL;
+    klass->set_governor = NULL;
+
+    /* Porperties */
+    g_object_class_install_property (object_class,
+                                     PROP_CPU,
+                                     g_param_spec_uint ("cpu",
+                                                        NULL,
+                                                        NULL,
+                                                        0,
+                                                        G_MAXUINT,
+                                                        0,
+                                                        G_PARAM_CONSTRUCT_ONLY |
+                                                        G_PARAM_READWRITE));
 }
 
 static void
 cpufreq_selector_set_property (GObject      *object,
-			       guint         prop_id,
-			       const GValue *value,
-			       GParamSpec   *spec)
+                               guint         prop_id,
+                               const GValue *value,
+                               GParamSpec   *spec)
 {
-	CPUFreqSelector *selector = CPUFREQ_SELECTOR (object);
-
-        switch (prop_id) {
-        case PROP_CPU:
-                selector->priv->cpu = g_value_get_uint (value);
-                break;
-        default:
-                G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
-		break;
-        }
+    CPUFreqSelector *selector = CPUFREQ_SELECTOR (object);
+
+    switch (prop_id) {
+    case PROP_CPU:
+        selector->priv->cpu = g_value_get_uint (value);
+        break;
+    default:
+        G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
+        break;
```

Comment:
default doesn't require the break statement

Reply:
Fixed here and at line 125.